script-library/docker-debian.sh

```diff
@@ -17,6 +17,9 @@ USE_MOBY=${5:-"true"}
 DOCKER_VERSION=${6:-"latest"}
 DOCKER_DASH_COMPOSE_VERSION=${7:-"v1"} # v1 or v2
 MICROSOFT_GPG_KEYS_URI="https://packages.microsoft.com/keys/microsoft.asc"
+DOCKER_DASH_COMPOSE_VERSION="1"
```

Comment:
I believe this line was removed in the previous commit.

Or is the pinning back to v1 intentional?

Reply:
Ah! Thank you for catching this - removed.